public/app/features/playlist/PlaylistForm.tsx

```diff
@@ -18,16 +18,22 @@ interface Props {
 }
 
 export const PlaylistForm = ({ onSubmit, playlist }: Props) => {
+  const [saving, setSaving] = useState(false);
   const { name, interval, items: propItems } = playlist;
   const tagOptions = useMemo(() => {
     return () => getGrafanaSearcher().tags({ kind: ['dashboard'] });
   }, []);
 
   const { items, addById, addByTag, deleteItem, moveItem } = usePlaylistItems(propItems);
 
+  const doSubmit = (list: Playlist) => {
+    setSaving(true);
```

Comment:
Do we ever set saving to false? May not be necessary though as in testing locally it appears on both success and error user is redirected to main playlist page

Reply:
the parent `onChange` always goes back to the root 🤷